packages/plugin-core/src/test/suite-integ/AddAndCommitCommand.test.ts

```diff
@@ -190,6 +113,7 @@ suite("GIVEN Workspace Add And Commit command is run", function () {
         ConfigUtils.setWorkspaceProp(config, "workspaceVaultSyncMode", "sync");
         return config;
       },
+      timeout: 1e6,
```

Comment:
i think 1e6 is useful for debugging but we don't actually expect the test to take this long, right? can we adjust this to a 10s timeout (1e4) or something closer to the expected latency?